src/assets/data/oil-refining.json

```diff
@@ -67,34 +67,30 @@
         "simpleLiquefactionCrackingRatio": [
             {
                 "iconId": "Coal_liquefaction",
-                "count": 8
+                "count": 12
             },
             {
                 "iconId": "Heavy_oil_cracking",
-                "count": 1
+                "count": 8
             },
             {
                 "iconId": "Light_oil_cracking",
-                "count": 3
+                "count": 11
             }
         ],
         "perfectLiquefactionCrackingRatio": [
             {
                 "iconId": "Coal_liquefaction",
-                "count": 25
+                "count": 60
             },
             {
                 "iconId": "Heavy_oil_cracking",
-                "count": 3
+                "count": 39
             },
             {
                 "iconId": "Light_oil_cracking",
-                "count": 9
+                "count": 55
             },
```

Comment:
extra comma i think
data is not loading, and there is a console error
https://deploy-preview-118--factoriocheatsheet.netlify.com/

Reply:
I've removed the comma at the end of the section.